app/task_routes.py

```diff
@@ -0,0 +1,122 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify, abort
+import datetime
+import os
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+# create helper function to post message to Slack
+def slack_bot(slack_message):
+    path = "https://slack.com/api/chat.postMessage"
+    SLACK_API_KEY = os.environ.get("SLACK_API_KEY")
+    HEADERS = {"Authorization": SLACK_API_KEY}
```

Comment:
Since your .env file isn't checked into Github (which is the right way to go about keeping your secrets secret), I'm assuming the file has "Bearer 384242somerandomapikey39482" in it, which works fine.

However, the API key could be used elsewhere or in places where you don't want the word "Bearer" before it, so it's best to use an f-string to add the "Bearer" and leave the environment variable as just the actual API key.

```python
"Authorization": f"Bearer {SLACK_API_KEY}"
```